backend/internal/service/handler/users/user.go

```diff
@@ -99,7 +111,7 @@ func (h *Handler) CalculateScore(c *fiber.Ctx) error {
 
 func (h *Handler) GetUserProfile(c *fiber.Ctx) error {
```

Comment:
seems like a code smell if you switch on flag and have different underlying implementations. to me, this suggests they should be separate endpoints.

Reply:
yes spoke to Dessy about this yesterday and decided to get rid of flag